cegs_portal/search/templates/search/v1/experiment_list.html

```diff
@@ -1,39 +1,140 @@
-{% extends "base.html" %}
-{% load static i18n %}
+{% extends "base.html" %} {% load static i18n %} {% block css %} {{ block.super }}
+<style>
+    .faded-line {
```

Comment:
I don't think this style is used, you can remove it.

Reply:
Oh ok. You were referring to this one on the experiment list page. Removed!